tools/upgraders.js

```diff
@@ -227,6 +227,12 @@ the guide about breaking changes here:`,
     packagesFile.writeIfModified();
   },
 
+  "1.3.0-census-package": function (projectContext) {
```

Comment:
Just wanted to double check that this code will only ever try to add the package once - i.e if the user updates to 1.3, removes the package, then updates to 1.3.1 - it's not going to add the package again. Is that correct?